build.gradle

```diff
@@ -285,12 +286,19 @@ dependencies {
   implementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
 
+  implementation group: 'com.github.tomakehurst', name: 'wiremock', version: '2.27.2'
```

Comment:
I doubt WireMock is being used in implementation? Can we move this to the appropriate scope please

Reply:
removed